apps/api/src/applicant/entity/ienapplicant.entity.ts

```diff
@@ -52,7 +52,7 @@ export class IENApplicant {
   @Column('varchar', { nullable: true })
   pr_status?: string;
 
-  @Column('jsonb', { nullable: true })
+  @Column('jsonb', { nullable: false, default: `[]` })
```

Comment:
In which case does it cause a problem? considering nursing_educations is optional, I think null shouldn't be an issue.